### ISSUE TITLE
MinUV mucking up bandpass solutions

### ISSUE THREAD

**Issue by AlecThomson** [tjgalvin/flint#49]
Beware, me hearties - there be some monsters at sea. We've found some wonky bandpass solutions using a minUV parameter of "200" in AOcalibrate. This presented itself as a smooth ripple in the Q/U cubes. Through investigating the AOcalibrate solutions, we found the same ripple in the X/U bandpass amplitudes:
![image](https://github.com/tjgalvin/flint/assets/9074527/8df75685-698c-4871-b0f6-a076ccc90264)

After reprocessing with a higher cutoff of "500", this issue was resolved:
![image](https://github.com/tjgalvin/flint/assets/9074527/21cf91a5-699d-4e97-9dea-eb3a41c2db99)

A few notes:

- The bandpass solutions are being impacted by short-baseline interference. The Sun is a likely suspect.
- AOcalibrate appears to use UVdist in wavelengths. i.e. "500" is 500 $\lambda$
- The ASKAPsoft pipeline uses a cutoff in metres of 200m by default. For reference, 200m at 888MHz is about 592 $\lambda$

To avoid this in future we need a couple of things:

- [x] #50
- [x] #51

**Comment by tjgalvin** [comment]
Yes -- I think this just goes to my misunderstand of what `calibrate -minuv` actually does. 

The set of rows dropped within `calibrate` are done:

```c
if((_minUVW >= 0.0 && (u*u + v*v + w*w < _minUVW*_minUVW))||(_maxUVW>=0.0 && (u*u + v*v + w*w > _maxUVW*_maxUVW)))
    selected = false;
```

which means that the `-minuv` is specifying the cutoff in lambda. i did not appreciate this when I was transplanting the code. Silly pirate that I be, arrr. 

For the moment I have bumped value up to `500`, but ideally it should be varied automatically based on the maximum (or nominal?) frequency in the measurement set.

**Comment by tjgalvin** [comment]
Actually, upon reflect I do think the argument is in meters. I just misunderstood things, like a silly pirate. It just needs to be bump up from 200 to something higher.

**Comment by tjgalvin** [comment]
I have a branch addressing these points under `caibator-validation`. 

I think I should try to add an option to opt into attempting to upload the image artifacts to prefect, i.e. enable the functionality via a CLI argument.

**Comment by tjgalvin** [comment]
I believe these have been addressed in #52 , which has been merged.
